src/Microsoft.Extensions.PlatformAbstractions/Internal/PlatformApis.cs

```diff
@@ -2,19 +2,24 @@
 using System.IO;
 using System.Runtime.InteropServices;
 
-namespace Microsoft.Extensions.PlatformAbstractions.Native
+namespace Microsoft.Extensions.PlatformAbstractions.Internal
 {
-    internal static class PlatformApis
+    public static class PlatformApis
     {
-        private class DistroInfo
-        {
-            public string Id;
-            public string VersionId;
-        }
-
         private static readonly Lazy<Platform> _platform = new Lazy<Platform>(DetermineOSPlatform);
         private static readonly Lazy<DistroInfo> _distroInfo = new Lazy<DistroInfo>(LoadDistroInfo);
 
+        public static string GetArch()
+        {
+#if NET451
+            return Environment.Is64BitProcess ? "x64" : "x86";
+#elif NETSTANDARD1_3
+            return RuntimeInformation.ProcessArchitecture.ToString().ToLowerInvariant();
+#else
+            return IntPtr.Size == 8 ? "x64" : "x86";
```

Comment:
It was already a problem with the code before. I looked in to P/Invokes to solve it but didn't think it was worth the effort right now.